boards/arm/actinius_icarus/dts/bindings/actinius,charger-enable.yaml

```diff
@@ -0,0 +1,26 @@
+# Copyright (c) 2022 Actinius B.V.
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+  The Icarus IoT Board v2 provides the user with an option to disable the
+  charger or to leave it on auto. The option can be changed by changing
+  the `charger` property in the associated dt node to `auto`
+  or `disabled`.
+
+compatible: "actinius,charger-enable"
+
+include: base.yaml
+
+properties:
+    gpios:
+      type: phandle-array
+      required: true
+      description: Pin used to enable/disable the charger
+
+    charger:
```

Comment:
If there are only 2 states like this, wouldn't a boolean make more sense?

Reply:
Technically it can be a bool; would that be preferable? In my opinion the enums were more fitting because the charger can be disabled, otherwise it has it's own logic about when it is on (i.e. auto). Let me know what you think, thanks